v1/brokers/kafka/kafka.go

```diff
@@ -0,0 +1,298 @@
+package kafka
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"sync/atomic"
+
+	"github.com/RichardKnop/machinery/v1/brokers/errs"
```

Comment:
Please organize imports in 3 different groups:

- Library
- 3rd party
- Local

Same as described here: https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e